README.md

```diff
@@ -98,23 +100,48 @@ This will copy the GitHub repository contents into a new folder in the current d
 cd oot
 ```
 
-#### 3. Prepare a base ROM
+#### 3. Install python dependencies
 
-Copy over your copy of the Master Quest (Debug) ROM inside the root of this new project directory.
-Rename the file to "baserom_original.z64", "baserom_original.n64" or "baserom_original.v64", depending on the original extension.
+The build process has a few python packages required that are located in `requirements.txt`.
 
-#### 4. Setup the ROM and build process
+It is recommend to setup a virtual environment for python to localize all dependencies. To create a virtual environment:
+
+```bash
+python3 -m venv .venv
+```
+
+To start using the virtual environment on your current terminal run:
+
+```bash
+. .venv/bin/activate
+```
+
+Keep in mind for each new terminal session you'll need to **activate** the Python virtual environment again. There's no need to create the virtual environment again.
+
+Now you can install the Python dependencies, to do so run:
+
+```bash
+python3 -m pip install -U -r requirements.txt
+```
+
+#### 4. Prepare a base ROM
+
+Copy over your copy of the Master Quest (Debug) ROM inside the `baseroms/gc-eu-mq-dbg/` path of this new project directory.
+
+Rename the file to `baserom.z64`, `baserom.n64` or `baserom.v64`, depending on the original extension.
+
+#### 5. Setup the ROM and build process
 
 Setup and extract everything from your ROM with the following command:
 
 ```bash
 make setup
 ```
 
-This will generate a new ROM called "baserom.z64" that will have the overdump removed and the header patched.
+This will generate a new ROM called "baseroms/gc-eu-mq-dbg/baserom-uncompressed.z64" that will have the overdump removed and the header patched.
```

Comment:
```suggestion
This will generate a new ROM "baseroms/gc-eu-mq-dbg/baserom-uncompressed.z64" that will have the overdump removed and the header patched.
```